package.json

```diff
@@ -38,7 +39,8 @@
     "commander": "^7.2.0",
     "cross-env": "^7.0.3",
     "js-yaml": "^3.14.1",
-    "randexp": "^0.5.3"
+    "randexp": "^0.5.3",
+    "run-script": "^0.1.1"
```

Comment:
@NektariosFifes not sure why you use this. But be careful when using dependencies with such a low activitiy.

Easy to introduce vulnerabilities